ibexa/oss/3.3/templates/themes/standard/pagelayout.html.twig

```diff
@@ -7,7 +7,7 @@
         {% set title = ez_content_name( content ) %}
     {% endif %}
     <title>{{ title|default( 'Home' ) }}</title>
-    <meta name="generator" content="Ibexa DXP"/>
+    <meta name="generator" content="Ibexa Platform"/>
```

Comment:
```suggestion
    <meta name="generator" content="Ibexa Open Source"/>
```

Reply:
Fixed in https://github.com/ibexa/recipes/pull/15/commits/db5e249dd30e0a00d77097f8d08aa7f2736588dd